ui-v2/app/adapters/policy.js

```diff
@@ -21,17 +21,21 @@ export default Adapter.extend({
       ${{ index }}
     `;
   },
-  requestForCreateRecord: function(request, data) {
+  requestForCreateRecord: function(request, serialized, data) {
     return request`
       PUT /v1/acl/policy?${{ [API_DATACENTER_KEY]: data[DATACENTER_KEY] }}
+
+      ${serialized}
     `;
   },
-  requestForUpdateRecord: function(request, data) {
+  requestForUpdateRecord: function(request, serialized, data) {
     return request`
       PUT /v1/acl/policy/${data[SLUG_KEY]}?${{ [API_DATACENTER_KEY]: data[DATACENTER_KEY] }}
+
+      ${serialized}
     `;
   },
-  requestForDeleteRecord: function(request, data) {
+  requestForDeleteRecord: function(request, serialized, data) {
```

Comment:
This wouldn’t need to happen in this PR, but seeing how all these adapters had to be updated in very similar ways and how create/update/delete all seem to follow the same pattern in most of the adapters, I wonder if you could generalise using something like [`BuildURLMixin`](https://api.emberjs.com/ember-data/release/classes/BuildURLMixin)?